buku.py

```diff
@@ -591,7 +601,7 @@ def delete_tag_at_index(self, index, tags_in):
         tags_to_delete = tags_in.strip(DELIM).split(DELIM)
 
         if index == 0:
-            resp = input('Delete specified tag(s) from ALL bookmarks? (y/n): ')
+            resp = read_in('Delete specified tag(s) from ALL bookmarks? (y/n): ')
```

Comment:
`resp = read_in('Delete specified tag(s) from ALL bookmarks? (y/n): ')`

please change it to:

`resp = read_in('Delete the tag(s) from ALL bookmarks? (y/n): ')`